File: tournaments/toDecimal/toDecimal.js
function toDecimal(k, n) {

    var result = 0;
    var power = 1;

    for (var i = n.length - 1; i >= 0; i--) {
        result += (n[i] - '0') * power;
        power *= k;
    }
  
    return result;
}  
  

File: tournaments/toDecimal/toDecimal.cpp
int toDecimal(int k, std::string n) {

  int result = 0;
  int power = 1;
  for (int i = (int)n.size() - 1; i >= 0; i--) {
    result += int(n[i] - '0') * power;
    power *= k;
  }

  return result;
}
